.github/workflows/CI.yaml

```diff
@@ -54,25 +54,13 @@ jobs:
       - name: Refine test environment
         shell: bash -l {0}
         run: |
-          # Install openmm-forcefields via pip (until available via omnia)
-          conda install --yes tinydb "openforcefield>=0.6.0" "openforcefields>=1.0.0"
-          pip install https://github.com/openmm/openmm-forcefields/archive/master.tar.gz
           case ${{ matrix.cfg.openmm }} in
             latest)
               echo "Using latest release OpenMM."
-              conda install --quiet -c omnia openmm;;
-            rc)
```

Comment:
I would leave the `rc` branch here, but commented out, because at some point we will have `conda-forge/label/openmm-rc` packages for the new versions, and we would benefit a lot from these downstream tests.